File: WebService/src/app/model/flights/flight.result.dto.ts
import { FlightItineraryDto } from "./flight.itinerary.dto";

export class FlightResultDto {
  constructor(
    public id: string,
    public flightCode: string,
    public departureDayItineraries: FlightItineraryDto[],
    public returnDayItineraries: FlightItineraryDto[],
    public price: number,
    public currency: string,
    public priceWithFare: number
  ) {}
}

File: Client/Models/Flights/search/FlightResultDto.cs
namespace Client.Models.Flights.search
{
    public class FlightResultDto
    {
        public string id { get; set; }

        public List<FlightItineraryDto> departureDayItineraries { get; set; }
        public List<FlightItineraryDto> returnDayItineraries { get; set; }

        public double price { get; set; }
        public string currency { get; set; }

        public double priceWithFare { get; set; }
    }
}
